README.adoc

```diff
@@ -252,9 +231,11 @@ Results :
 Tests run: 3, Failures: 0, Errors: 0, Skipped: 0
 ----
 
-To see if the tests detect a failure, comment out the `changeSystemProperty()` methods
-in the `FaultToleranceTest.java` file. Rerun the Maven build.
+To see if the tests detect a failure, comment out the `@Fallback` annotation
+in the `src/main/java/io/openliberty/guides/inventory/InventoryManager.java` file. Rerun the Maven build.
 You see a test failure occur for the `testFallbackForGet()` test case.
+The test failure indicates a 500 HTTP response code because the fallback mechanism was not in place to action
+when the IOException was thrown.
```

Comment:
We can likely make this sentence present tense. Can we also find a more common word for "to action," such as "to run" or "to start"? (I usually see "action" as a noun.)

The test failure indicates a 500 HTTP response code because the fallback mechanism is not in place to run when the IOException was thrown.

Reply:
Note from my grammar checker about the word "thrown": "Use only in documentation of programming languages (such as Java, C++, Ruby, Lisp or others) that support throw and catch programming blocks, as in "throw an exception." Otherwise, use an alternative construction such as "produce an error.""
I think it's OK here, but, once again, I'll let you decide.